examples/hurricanes/main.js

```diff
@@ -369,8 +369,8 @@ $(function () {
     }
   );
 
-  // Create a feature layer to draw on
-  layer = map.createLayer('feature');
+  // Create a feature layer to draw on.
+  layer = map.createLayer('feature', {features: ['line.multicolor']});
```

Comment:
Our gl lines can vary color from point to point.  The d3 lines can't.

We probably should document the feature capabilities in the base feature.